project.clj

```diff
@@ -1,4 +1,4 @@
-(def tk-version "1.3.0")
+(def tk-version "1.3.1-SNAPSHOT")
```

Comment:
And it probably makes sense to hold off on the TK release until the current slew of tickets is resolved, so this may need to sit a little longer.